recovery-lab.md

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
I removed the suggestion @kuronosec did just to have it as clean as possible but I agree with him, this lab is not clear, what would be done with this lab?

Reply:
@xavier506 @kuronosec what do you suggest?

Reply:
@leisterfrancisco maybe we can ask the student to run nodeos and then shutdown it ungracefully, e.g. with `kill -kill $(pidof nodeos)`. And then recover from blocks.log, snapshot or backup.

Reply:
Yes, that sounds good. I'm going to add this as an issue to work on it later. What do you think @xavier506?

Reply:
sounds good